src/widgets/AppBar.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { StackHeaderProps } from "@react-navigation/stack";
+import { Appbar } from "react-native-paper";
+
+import MenuButton from "./MenuButton";
+
+export default function AppBar(props: StackHeaderProps & { menuFallback: boolean }) {
+  const { insets, menuFallback, navigation, previous, scene } = props;
+  const { options } = scene.descriptor;
+  const title = (options.headerTitle)
```

Comment:
Please replace this with:
```
const title = options.headerTitle ?? options.title ?? scene.route.name;
```
This is very confusing.

Reply:
Also, why is it needed all of a sudden? It wasn't there before.

Reply:
We didn't need it before because it's handled by default by React Navigation's Header, but we need to do it manually when using the one from React Native Paper.

Reply:
Yeah, that's what I figured. Interesting that there's no wrapper to do it. Anyhow, other comment applies.